README.md

```diff
@@ -51,12 +51,12 @@ Whenever an aliased CyHy command is executed, it will use the current working di
 
 ### Advanced configuration
 
-By default, the container will look for your CyHy configurations in `/etc/cyhy`.  This location can be changed by setting the `CYHY_CONF_DIR` environment variable to point to your CyHy configuration directory.  The commands will also attempt to run using the `cisa/cyhy-core` image.  A different image can be used by setting the `CYHY_CORE_IMAGE` environment variable to the image name.
+By default, the container will look for your CyHy configurations in `/etc/cyhy`.  This location can be changed by setting the `CYHY_CONF_DIR` environment variable to point to your CyHy configuration directory.  The commands will also attempt to run using the `cisagov/cyhy-core` image.  A different image can be used by setting the `CYHY_CORE_IMAGE` environment variable to the image name.
 
 Example:
 ```
```

Comment:
I think this should have `bash` formatting.

Reply:
Thanks - nice catch!  Added in https://github.com/cisagov/cyhy-core/pull/50/commits/0e7619fc563f67eb9d933bb0e358983165428a29.